add supporters.json (#145)
BlackPythonDevs/blackpythondevs: policies/supporters_reporting.md

---
name: Supporters Reporting
created: 1 Jan 2025
updated: 1 Jan 2025
created_by: Jay Miller
last_updated_by: Jay Miller
---

> ## A record of supporters to be kept

Reviewer: ```suggestion
# Record of Supporters
```

> A public record of supporters by year should be kept. This allows us to see an individual's contribution year over year.
> 
> In order to protect the privacy of individuals, No information other than their name shall be made public.

Reviewer: ```suggestion
We keep an annual record of our supporters to track their contributions over time. To protect their privacy, we only share their names in public.
```

> ## Update/Removal of individuals

Reviewer: ```suggestion
## Updating or Removing Supporter Information from Public Records
```

> Upon request in writing (via email to leadership@blackpythondevs.com), individuals can have one or more instances of their name updated or removed from the public record files. Considering the file is kept in version control, information cannot be completely deleted. To not have your name included please mark to be anonymous at time of donation.
> 
> Record keeping is done using [CommitChange](https://commitchange.com). Information in CommitChange can be updated but should not be removed.

Reviewer: ```suggestion
You can request updates or removal of your name from our public records by sending a written request via email to **leadership@blackpythondevs.com**.

However, since we manage these records using version control, we cannot completely delete all information. So if you prefer not to have your name appear in our public records, please mark it as `anonymous` at the time of your donation.

We use [CommitChange](https://commitchange.com) for record-keeping. While we can update information in `CommitChange`, we cannot remove it entirely.
```

> ## Adding a name to the list of Supporters

Reviewer: ```suggestion
## Adding a Name to the List of Supporters
```

Author: This is an interesting suggestion as some markdown CLI tools will want you to not capitalize the entire header. 

https://www.scribbr.com/academic-writing/capitalization-titles-headings/

I traditionally go with you title-case H1 but only the first letter for H2 and down.

Reviewer: I didn’t realize that some CLI tools require headings not to be fully capitalized. However, I believe it’s important to maintain consistency in formatting. Perhaps we should consider using sentence case for all headings instead and apply this approach throughout the entire documentation. This might also be a good opportunity to create a style guide if we don’t already have one.

> The supporters list will be updated at the end of the year using the end-of-year export feature in CommitChange.

Reviewer: ```suggestion
The supporters’ list is annually updated using the end-of-year export feature in `CommitChange`.
```

> ## Foundational Supporters listed on the website

Reviewer: ```suggestion
## Foundational Supporters
```

Author: all that I said about capitalization and I'm inconsisten... LOL

> If a donor donates over $200 in one calendar year, their name will be added to the website as a foundational supporter. This information is updated monthly using [CommitChange](https://commitchange.com) our payment processor.
> 
> Foundational Supporters are listed from the month after they reach $200 donated in that calendar year and the following year.

Reviewer: ```suggestion
Donors who contribute over $200 in a calendar year will be recognized as Foundational Supporters, and their names will be featured on our website. This information is updated monthly through our payment processor, `CommitChange`.

Foundational Supporters are listed starting from the month they reach $200 in donations for that calendar year and remain listed through the following year.
```
